src/styles/blocks/card.scss

```
@@ -0,0 +1,62 @@
+.card {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.card__container {
+  display: flex;
+  position: relative;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+  color: $txtbasis;
+}
+
+.card__article {
+  font-size: 10px;
+  line-height: 14px;
+  color: $txtanother;
+  margin-right: 65px;
+  padding-top: 4px;
```

margin (to make outer indent between elements)
```suggestion
  margin-top: 4px;
```